.ddev/commands/web/gulp

```diff
@@ -1,7 +1,28 @@
 #!/bin/bash
 
+set -e
+
 ## Description: Run robo commands inside the web container for the client app.
 ## Usage: gulp
 ## Example: "ddev gulp publish" or "ddev gulp".
 
+CLIENT_PATH="../../client"
+
+if [[ -z "${EHEZA_INFRA_REPO_REMOTE}" ]]; then
```

Comment:
@AronNovak shouldn't we do all this only when running `gulp publish`?

Reply:
good point, added a condition for that